nautobot_netbox_importer/__init__.py

```diff
@@ -22,6 +22,7 @@ class NautobotNetboxImporterConfig(PluginConfig):
     description = "Data importer from NetBox 2.10.x to Nautobot."
     base_url = "netbox-importer"
     required_settings = []
+    min_version = "1.0.0b2"
```

Comment:
Will want to bump this to `1.0.0b3` as soon as we actually have that as a Nautobot version number.